### PR TITLE
Fix misspelled brand names

### LINKED ISSUE #30826: Misspelled brand names in content
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/.github/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

* content/actions/hosting-your-own-runners/managing-self-hosted-runners-with-actions-runner-controller/deploying-runner-scale-sets-with-actions-runner-controller.md
* content/code-security/code-scanning/creating-an-advanced-setup-for-code-scanning/codeql-code-scanning-for-compiled-languages.md
* content/get-started/writing-on-github/working-with-advanced-formatting/attaching-files.md
* contributing/codespace.md
* (content using the reusable: data/features/actions-default-workflow-permissions-restrictive.yml)

### What part(s) of the article would you like to see updated?

references to brands should be spelled correctly

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by jsoref**
### Why:

Closes: #30826

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [ ] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

### CODE REVIEW

File: content/actions/hosting-your-own-runners/managing-self-hosted-runners-with-actions-runner-controller/deploying-runner-scale-sets-with-actions-runner-controller.md [hunk 1]
```diff
@@ -574,7 +574,7 @@ When you set the `containerMode` in the `values.yaml` file for the [`gha-runner-
 
 Depending on which value you set for the `containerMode`, a configuration will automatically be injected into the `template` section of the `values.yaml` file for the `gha-runner-scale-set` helm chart.
 - See the [`dind` configuration](https://github.com/actions/actions-runner-controller/blob/5347e2c2c80fbc45be7390eab117e861d30776d1/charts/gha-runner-scale-set/values.yaml#L110).
-- See the [`kuberbetes` configuration](https://github.com/actions/actions-runner-controller/blob/5347e2c2c80fbc45be7390eab117e861d30776d1/charts/gha-runner-scale-set/values.yaml#L160).
+- See the [`kubernetes` configuration](https://github.com/actions/actions-runner-controller/blob/5347e2c2c80fbc45be7390eab117e861d30776d1/charts/gha-runner-scale-set/values.yaml#L160).
```
Comment (jsoref) [inline content/actions/hosting-your-own-runners/managing-self-hosted-runners-with-actions-runner-controller/deploying-runner-scale-sets-with-actions-runner-controller.md:577]: https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners-with-actions-runner-controller/deploying-runner-scale-sets-with-actions-runner-controller
image

https://docs-30827-2eb9b9.preview.ghdocs.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners-with-actions-runner-controller/deploying-runner-scale-sets-with-actions-runner-controller#customizing-container-modes
image

File: content/code-security/code-scanning/creating-an-advanced-setup-for-code-scanning/codeql-code-scanning-for-compiled-languages.md [hunk 2]
```diff
@@ -102,7 +102,7 @@ On Linux and macOS, the `autobuild` step reviews the files present in the reposi
 1. If none are found, search subdirectories for a unique directory with a build system for C/C++.
 1. Run an appropriate command to configure the system.
 
-For self-hosted runners, you will likely need to install the `gcc` compiler, and specific projects may also require access to `clang` or `mscv` executables. You will also need to install the build system (for example `msbuild`, `make`, `cmake`, `bazel`) and utilities (such as `python`, `perl`, `lex`, and `yacc`) that your projects depend on.
+For self-hosted runners, you will likely need to install the `gcc` compiler, and specific projects may also require access to `clang` or `msvc` executables. You will also need to install the build system (for example `msbuild`, `make`, `cmake`, `bazel`) and utilities (such as `python`, `perl`, `lex`, and `yacc`) that your projects depend on.
```
Comment (jsoref) [inline content/code-security/code-scanning/creating-an-advanced-setup-for-code-scanning/codeql-code-scanning-for-compiled-languages.md:105]: https://docs.github.com/en/code-security/code-scanning/creating-an-advanced-setup-for-code-scanning/codeql-code-scanning-for-compiled-languages#autobuild-for-cc
image
https://docs-30827-2eb9b9.preview.ghdocs.com/en/code-security/code-scanning/creating-an-advanced-setup-for-code-scanning/codeql-code-scanning-for-compiled-languages#autobuild-for-cc
image

File: content/get-started/writing-on-github/working-with-advanced-formatting/attaching-files.md [hunk 3]
```diff
@@ -78,7 +78,7 @@ We support these files:
 {%- endif %}
 - Log files (_.log_)
 - Markdown files (_.md_)
-- Microsoft Word (_.docx_), Powerpoint (_.pptx_), and Excel (_.xlsx_) documents
+- Microsoft Word (_.docx_), PowerPoint (_.pptx_), and Excel (_.xlsx_) documents
```
Comment (jsoref) [inline content/get-started/writing-on-github/working-with-advanced-formatting/attaching-files.md:81]: #### before
https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/attaching-files
image
#### after

https://docs-30827-2eb9b9.preview.ghdocs.com/en/get-started/writing-on-github/working-with-advanced-formatting/attaching-files
image

File: contributing/codespace.md [hunk 4]
```diff
@@ -1 +1 @@
-The content in the `contributing` directory has been deprecated. See "[Working on a GitHub Docs in a codepsace](https://docs.github.com/en/contributing/setting-up-your-environment-to-work-on-github-docs/working-on-github-docs-in-a-codespace)" in the GitHub Docs for the maintained version of this article.
+The content in the `contributing` directory has been deprecated. See "[Working on a GitHub Docs in a codespace](https://docs.github.com/en/contributing/setting-up-your-environment-to-work-on-github-docs/working-on-github-docs-in-a-codespace)" in the GitHub Docs for the maintained version of this article.
```
Comment (jsoref) [inline contributing/codespace.md:1]: image

File: data/features/actions-default-workflow-permissions-restrictive.yml [hunk 5]
```diff
@@ -1,5 +1,5 @@
 # Reference: #9014.
-# Versioning for enterprise/organization/repository policy settings for workflow permissions granted to GTIHUB_TOKEN to be readonly by default and not allow GitHub Actions to create or approve pull requests.
+# Versioning for enterprise/organization/repository policy settings for workflow permissions granted to GITHUB_TOKEN to be readonly by default and not allow GitHub Actions to create or approve pull requests.
```
Comment (jsoref) [inline data/features/actions-default-workflow-permissions-restrictive.yml:2]: This can't be seen in a preview as it's a comment

**Comment by nguyenalex836** [comment]
@jsoref Thanks so much for opening a PR! I'll get this triaged for review ✨

**Review by subatoi — APPROVED** [review]
Thanks for catching these! I'll mark this as ready to merge.

**Pull request merged**